home-application/src/main/java/com/softserveinc/ita/homeproject/application/apiservice/UserApiServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package com.softserveinc.ita.homeproject.application.apiservice;
+
+import com.softserveinc.ita.homeproject.api.NotFoundException;
+import com.softserveinc.ita.homeproject.api.UsersApiService;
+import com.softserveinc.ita.homeproject.model.CreateUser;
+import com.softserveinc.ita.homeproject.model.UpdateUser;
+import com.softserveinc.ita.homeproject.service.impl.UserServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Service;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+
+@Service
+@RequiredArgsConstructor
+public class UserApiServiceImpl implements UsersApiService {
+
+    private final UserServiceImpl userServiceImpl;
```

Comment:
Please use the interface instead of implementation.

Reply:
...

Reply:
Don't really understant the issue, sorry(
I will try to find more after daily meeting